main/guides/coreeval/permissions.md

```diff
@@ -0,0 +1,89 @@
+# Coding Permissions
+
+Write a json file with the permissions that the proposal will need. For example, [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm-permit.json) is the proposal Agoric created for the PSM contract:
```

Comment:
_comment_:
The example is somewhat out of date with the code on master. Probably not a big deal.

Reply:
I definitely think that's fine. The additional changes are contract-specific. However, one of the changes appears to be upgrade-related, so after I document the upgrade API, I want to return to this section, update the permissions page, and discuss the additional properties.